requirements-optional-linux.txt

```diff
@@ -3,6 +3,6 @@ pypotrace
 pyclipr
 meerk40t-barcodes
 numba
-scipy>=1.12.0,<=1.14.0
```

Comment:
This restriction is intentional as higher versions of scipy cause other problems. Please reinstate the restrictions